home/templates/home/question_detail.html

```diff
@@ -39,7 +39,7 @@ <h3> {{ question.title }} </h3>
 		<b>by {{ question.profile }} </b>
 	  </div>
 		<div class="question-body">
-		  <p class="question-content">{{ question.content }}</p>
+		  <p class="question-content">{{ question.content | safe }}</p>
```

Comment:
This is scary shit... you've effectively helped your users to perform script injection on your pages... if you want to treat the question content as safe you must have some validation **on the server side** that makes sure it only contains a small subset of HTML that you can actually trust.

Reply:
@ifireball added validation and tests for that

Reply:
after it will approve here I will do the same for #87